lib/matplotlib/animation.py

```diff
@@ -1777,3 +1781,106 @@ def _draw_frame(self, framedata):
 
             for a in self._drawn_artists:
                 a.set_animated(self._blit)
+
+class PlayerAnimation(FuncAnimation):
+    # inspired from https://stackoverflow.com/a/46327978/3949028
+    PLAY_SYMBOL = "$\u25B6$"
+    STOP_SYMBOL = "$\u25A0$"
+    PAUSE_SYMBOL = "$\u23F8$" #TODO use instead of STOP_SYMBOL, but doesn't work in Button.label
+    ONE_BACK_SYMBOL = "$\u29CF$"
+    ONE_FORWARD_SYMBOL = "$\u29D0$"
+
+    def __init__(self, func, init_func, min_value=0, max_value=100,
+                 pos=(0.125, 0.92), valstep=1, **kwargs):
```

Comment:
This is going to need a docstring including the required signatures of the functions passed in.